studio/components/interfaces/Support/Support.constants.ts

```diff
@@ -36,7 +36,7 @@ export const CATEGORY_OPTIONS = [
     query: undefined,
   },
   {
-    value: 'Problem',
+    value: 'Abuse',
```

Comment:
before we get this in, just need to double check on API's end whether this is a valid category

Reply:
yeah these are the available support categories so this change will send the support ticket to feedback instead

![image](https://github.com/supabase/supabase/assets/19742402/9bf78181-782b-4405-8278-4eaa4db1774d)

Reply:
while you're on this PR @saltcod would you mind just changing the values to using `SupportCategories` from shared-types? will help us prevent any sort of deviance between FE and API

but yeah these comments are separate to the issue that you're fixing in this PR, may need a different solution